tests/foreman/maintain/test_packages.py

```diff
@@ -273,22 +273,3 @@ def test_positive_fm_packages_update(request, sat_maintain):
     def _finalize():
         assert sat_maintain.execute('dnf remove -y walrus').status == 0
         sat_maintain.execute('rm -rf /etc/yum.repos.d/custom_repo.repo')
-
-
-@pytest.mark.stubbed
-def test_positive_fm_packages_sat_installer(sat_maintain):
-    """Verify satellite-installer is not executed after install/update
-    of satellite-maintain/rubygem-foreman_maintain package
-
-    :id: d73971a1-68b4-4ab2-a87c-76cc5ff80a39
-
-    :steps:
-        1. satellite-maintain packages install/update satellite-maintain/rubygem-foreman_maintain
-
-    :BZ: 1825841
-
-    :expectedresults: satellite-installer shouldn't be executed after install/update
-        of satellite-maintain/rubygem-foreman_maintain package
-
```

Comment:
I remember I've added this test in past, it verifies installer doesn't run after we install/update mentioned package, because of an entry in whitelist packages in /etc/yum/pluginconf.d/foreman-protector.whitelist
And IMO, this is unrelated to self-update or self-upgrade workflow, so it doesn't make sense to remove this test in favour of that test.

Reply:
This is an outdated way of updating the satellite-maintain package. We no longer tell customers to do this in our documentation. I don't see the need to have a stubbed case in our automation if users aren't supposed to do this. I am willing to at least remove the cherry-pick to other branches and just do this for master if you prefer that.

Reply:
Yeah, this isn't a documented way, it was before self-update or self-upgrade features existed iirc, so that's what BZ was about. We still support this way of update too and customers can try this anyway, so I think there's no harm if we keep this stubbed test OR this can be easily automated just by checking installer doesn't run after package update